.github/workflows/local-testnet.yml

```diff
@@ -70,3 +70,11 @@ jobs:
       - name: Stop local testnet with blinded block production
         run: ./stop_local_testnet.sh
         working-directory: scripts/local_testnet
+  local-testnet-success:
+    name: test-suite-success
+    runs-on: ubuntu-latest
+    needs: ["run-local-testnet"]
```

Comment:
Looks like we just need to depend on one "job" (according to actionlint at least). We'll see how it runs